hardhat.config.js

```diff
@@ -122,9 +122,18 @@ module.exports = {
     },
   },
 
-  solc: {
-    version: '0.6.6',
-    optimizer: require('./solcOptimiserSettings.js'),
+  solidity: {
+    compilers: [
+      {
+        version: "0.6.6",
+        settings: {
+          optimizer: {
+            enabled: true,
+            runs: 200
```

Comment:
Increase value to 430 pls

Reply:
Can't. Since I changed it to use IERC20Ext, using 430 goes beyond the bytecode size. No idea why.

Reply:
ohhhhhh

Reply:
I think the reason is **KyberNetwork** contract size is too big, right?

Maybe we should deploy KyberNetwork in this repo, instead of recompiling it.
This way we can increate optimizer runs.

Reply:
Maybe we resolve this issue in another PR